src/builder/edit_role.rs

```diff
@@ -1,7 +1,7 @@
 use std::collections::HashMap;
 
-use crate::internal::prelude::*;
 use crate::model::{guild::Role, Permissions};
+use crate::{internal::prelude::*, json::from_number};
```

Comment:
These should be separate `use`s.
```suggestion
use crate::internal::prelude::*;
use crate::json::from_number;
```